fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/metric/GenericMetricLoader.java

```diff
@@ -62,26 +60,17 @@ public void loadMetricDictionary(MetricDictionary metricDictionary) {
                 .forEach(dataSourceConfiguration -> {
                     dataSourceConfiguration.getMetrics()
                             .forEach(metric -> {
-                                ApiMetricName apiMetricName = new FiliApiMetricName(
-                                        metric,
-                                        dataSourceConfiguration.getValidTimeGrain()
+                                metrics.add(
+                                        new MetricInstance(
+                                                new LogicalMetricInfo(metric),
+                                                doubleSumMaker,
+                                                new DruidMetricName(metric)
+                                        )
```

Comment:
The constructor `new MetricInstance(apiMetricName, doubleSumMaker, fieldName)` has been deprecated in 2017. New constructor that takes a `LogicalMetricInfo` is used here.